recipes/glew/all/conanfile.py

```diff
@@ -0,0 +1,106 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+class GlewConan(ConanFile):
+    name = "glew"
+    description = "The GLEW library"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://github.com/nigels-com/glew"
+    topics = ("conan", "glew", "opengl", "wrangler", "loader", "binding")
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+    settings = "os", "arch", "build_type", "compiler"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    _source_subfolder = "source_subfolder"
+
+    def requirements(self):
+        self.requires("opengl/system")
+        self.requires("glu/system")
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("%s-%s" % (self.name, self.version), self._source_subfolder)
+        if "patches" in self.conan_data:
+            for patch in self.conan_data["patches"][self.version]:
+                tools.patch(**patch)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["BUILD_UTILS"] = "OFF"
+        cmake.configure()
+        return cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        for pdb_file in glob.glob(os.path.join(self.package_folder, "lib", "*.pdb")):
+            os.remove(pdb_file)
+
+        self.copy("include/*", ".", "%s" % self._source_subfolder, keep_path=True)
+        self.copy("%s/license*" % self._source_subfolder, dst="licenses",  ignore_case=True, keep_path=False)
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+
+        if self.settings.os == "Windows":
+            if self.settings.compiler == "Visual Studio":
+                if self.options.shared:
+                    if self.settings.build_type == "Release":
+                        self.copy("glew32.lib", "lib", "lib", keep_path=False)
```

Comment:
is it copying into the same folder? I feel like most of these lines are no longer needed as we use install, maybe we should just set up `cpp_info.bindis` or `cpp_info.libdirs`, but I think we probably should just package everything as install does, avoid redundant repackaging here